### PR TITLE
Prevented extensive use of the word "extensively".

### PR DESCRIPTION
**Pull request by Catchears**
Just a small change to the text as before most materials used "extensively" in their description.

### CODE REVIEW

**Comment by Catchears** [comment]
Test failed even though it compiled on my system. Closing.

**Comment by Anuken** [comment]
The tests are failing even on the current master branch, as I haven't fixed them yet.

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -1084,12 +1084,12 @@ tutorial.waves.mobile = The[lightgray] enemy[] approaches.\n\nDefend the core fo
 tutorial.launch = Once you reach a specific wave, you are able to[accent] launch the core[], leaving your defenses behind and[accent] obtaining all the resources in your core.[]\nThese obtained resources can then be used to research new technology.\n\n[accent]Press the launch button.
 
 item.copper.description = The most basic structural material. Used extensively in all types of blocks.
-item.lead.description = A basic starter material. Used extensively in electronics and liquid transportation blocks.
-item.metaglass.description = A super-tough glass compound. Extensively used for liquid distribution and storage.
+item.lead.description = A basic starter material. Used a lot in electronics and liquid transportation blocks.
```
Comment (Anuken) [inline core/assets/bundles/bundle.properties:1087]: "a lot" and "often" don't sound right; you could use a [synonym](https://www.thesaurus.com/browse/extensive).

Comment (Catchears) [reply core/assets/bundles/bundle.properties:1087]: Ok, I'll take a look.

**Comment by Catchears** [comment]
I'll take a look.

**Comment by sk7725** [comment]
Well isnt this an extensive Pull Request

**Comment by ILiekMelons** [comment]
<insert joke about extensiveness here>

**Comment by Anuken** [comment]
I'll handle this in the upcoming description rewrite.

**Pull request closed without merging**